encoding/protobuf/error.go

```diff
@@ -78,9 +83,9 @@ func GetErrorDetails(err error) []interface{} {
 	if err == nil {
 		return nil
 	}
-
-	if pberr, ok := err.(*pberror); ok {
-		return pberr.details
+	target := &pberror{}
```

Comment:
The `&pberror{..}` will be thrown away. Can do:
```suggestion
	var target *pberror
```

Reply:
Does this not set it to nil? Or is it okay to set it to nil?

Reply:
nil is okay because you're not passing nil to `errors.As` but the non-nil pointer to a nil pointer. Roughly:

```
var target *pberror  // nil
var toFill **pberror = &target // non-nil
errors.As(err, toFill)
  => *toFill = &pberror{...}
  => target  = &pberror{...}
```

So if original target is non-nil like in the current version, it'll get throw
away.

Reply:
Ah, I was confused because it was a pointer to a pointer. Makes sense.